src/actions/sortListActions.js

```diff
@@ -0,0 +1,33 @@
+import { SET_SORT } from './actionTypes';
+import { storage } from '../components/Dashboard/SortList/sortUtils';
+
+const setSort = (key, field, direction) => dispatch => {
+  storage.set(key, {
+    field,
+    direction,
+  });
+
+  dispatch({
+    type: SET_SORT,
+    key,
+    field,
+    direction,
+  });
+};
+
+const loadSort = key => dispatch => {
```

Comment:
This function is unnecessary. Please change the one above and use it for all the Redux communication